doc/api/modules.md

```diff
@@ -197,13 +197,14 @@ LOAD_NODE_MODULES(X, START)
 NODE_MODULES_PATHS(START)
 1. let PARTS = path split(START)
 2. let I = count of PARTS - 1
-3. let DIRS = [GLOBAL_FOLDERS]
+3. let DIRS = []
 4. while I >= 0,
    a. if PARTS[I] = "node_modules" CONTINUE
    b. DIR = path join(PARTS[0 .. I] + "node_modules")
    c. DIRS = DIRS + DIR
    d. let I = I - 1
-5. return DIRS
+5. DIRS = DIRS.concat(GLOBAL_FOLDERS)
+6. return DIRS
```

Comment:
Why not refactor these into one line? (or am I not getting the point)
```suggestion
5. return DIRS.concat(GLOBAL_FOLDERS)
```

Reply:
@bl-ue  I think it may be a bit more clear to read.

Reply:
Personally i find the one-liner far more clear to read than any reassignment would be. If it’s going to stay two lines, let’s use a new variable name?

Reply:
@ljharb Thanks,

I'd prefer one-liner. Will do tonight.